src/test/article.spec.js

```diff
@@ -213,3 +219,246 @@ describe('GET api/v1/articles', () => {
       });
   });
 });
+describe('POST/GET/PUT/DELETE ACTIVITIES ON ARTICLES CONTROLLER', () => {
+  const token = helper.jwtSigner({
+    id: 2,
+    username: 'Grace',
+    email: 'grace@grace.com'
+  });
+
+  describe('POST /api/v1/articles', () => {
+    beforeEach(async () => {
+      await db.Bookmark.destroy({ cascade: true, truncate: true });
+      await db.User.destroy({ cascade: true, truncate: true });
+      await db.Article.destroy({ cascade: true, truncate: true });
+    });
+    it('should create article successfully with valid user input', async () => {
+      const response = await chai
+        .request(app)
+        .post(ARTICLES_API)
+        .set('x-access-token', token)
+        .send(articleSample);
+      expect(response.status).to.equal(201);
+      expect(response.body.data).to.have.property('id');
+      expect(response.body.data.id).to.be.a('number');
+      expect(response.body.data).to.have.property('title');
+      expect(response.body.data.title).to.equal(articleSample.title);
+      expect(response.body.data).to.have.property('description');
+      expect(response.body.data.description).to.equal(
+        articleSample.description
+      );
+      expect(response.body.data).to.have.property('body');
+      expect(response.body.data.body).to.equal(articleSample.body);
+      expect(response.body.data).to.have.property('image');
+      expect(response.body.data.image).to.equal(articleSample.image);
+      expect(response.body.data).to.have.property('slug');
```

Comment:
Property assertion on Slug is good here because it is random, meaning you don't control it's value